### ISSUE TITLE
Inconsistent implementation of get_random_layer_generator for Torch and TF

### ISSUE THREAD

**Issue by aaarrti** [understandable-machine-intelligence-lab/Quantus#285]
### Description
- Here https://github.com/understandable-machine-intelligence-lab/Quantus/blob/main/quantus/helpers/model/tf_model.py#L272 for Keras model we shuffle each layer weights.
- Here https://github.com/understandable-machine-intelligence-lab/Quantus/blob/main/quantus/helpers/model/pytorch_model.py#L213 for torch model we reset module parameters (to default ones.)

### Minimum acceptance criteria
- Include both weight modification strategies for both framework's models.
- Keep the default version uniform.
